### PR TITLE
Add Danawa by Cowave

### PR DESCRIPTION
**Pull request by seia-soto**
refs https://github.com/yous/YousList/pull/255
refs http://www.ftc.go.kr/bizCommPop.do?wrkr_no=1178140065 (requires English captcha)

Danawa is a price comparison site like BestBuy. Unfortunately, they don't offer English version of website or privacy policy.

<details>
<summary>Their info from `ftc.co.kr` (Korean Fair Trade Commission)</summary>

Company Name:

```

business name | Connect Wave Co., Ltd.

```

Domains:

```
www.danawa.com
www.danawa.com hi-farm.com www.farmtail.net www.fivecolor.co.kr www.makeglob.com www.ssandomain.com www.prshop.com www.shopinside.net www.makefran.co.
www.playauto.co.kr www.plto.com www.playauto.freebill.co.kr
```

</details>

**Notes**

- `las.danawa.com` is only accessible through their mobile application, confirmed on iOS
- `ad.danawa.com` is for self-promtion

### CODE REVIEW

File: db/patterns/danawa.eno [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+website_url: https://danawa.com/
+organization: connectwave
+
+--- domains
+ad.danawa.com
+las.danawa.com
+--- domains
+
+--- filters
+||ad.danawa.com
```
Comment (philipp-classen) [inline db/patterns/danawa.eno:12]: If I go to the domain (https://ad.danawa.com/), it does an opt-out. Would be an argument in favor of triggering only for $3p requests (`||ad.danawa.com$3p`)

Comment (seia-soto) [reply db/patterns/danawa.eno]: Fixed.

File: db/organizations/connectwave.eno [hunk 2]
```diff
@@ -0,0 +1,9 @@
+name: Connectwave
+website_url: https://www.danawa.com/
+privacy_policy_url: https://www.danawa.com/info/helprule_private.html
+privacy_contact: privacy@cowave.kr
+country: KR
+description: "Danawa is a price comparison site like BestBuy."
```
Comment (philipp-classen) [inline db/organizations/connectwave.eno:6]: I found the following description. Do you think it is factually correct and makes sense to change to it as the company description?

```
ConnectWave Co Ltd, formerly DANAWA Co., Ltd, is a Korea-based company mainly engaged in the operation of online price comparison websites.
```

Comment (seia-soto) [reply db/organizations/connectwave.eno:6]: It seems like more descriptive than my one. I had no idea how to explain the website. :)

**Review by philipp-classen — APPROVED** [review]

**Pull request merged**